int32[] depth_jumps
float32[] range_date
int32 rotation
int32 liniear_x